src/spikeinterface/postprocessing/template_similarity.py

```diff
@@ -32,8 +45,8 @@ class ComputeTemplateSimilarity(AnalyzerExtension):
     def __init__(self, sorting_analyzer):
         AnalyzerExtension.__init__(self, sorting_analyzer)
 
-    def _set_params(self, method="cosine_similarity"):
-        params = dict(method=method)
+    def _set_params(self, method="cosine", max_lag_ms=0, support="dense"):
```

Comment:
do we plan a backcompatible stuff when "cosine_similarity" is given ?

Reply:
yep, let's do it